util/util.go

```diff
@@ -52,18 +58,44 @@ func Unmarshal(x starlark.Value) (val interface{}, err error) {
 
 			pval, err = Unmarshal(dictVal)
 			if err != nil {
+				err = errors.Wrap(err, "failed unmarhasl value")
```

Comment:
Let's use go 1.13 error wrapping instead:

```suggestion
				err = fmt.Errorf("unmarshaling starlark value: %w", err)
```